.github/workflows/lint-test.yml

```diff
@@ -151,6 +151,18 @@ jobs:
         pip install -U build
         python -m build
 
+    # run the libcst parsers and check for changes
+    - name: libcst codemod
+      env:
+        LIBCST_PARSER_TYPE: "native"
+      run: |
+        nox -s codemod -- run-all
+        if [ -n "$(git status --porcelain disnake/)" ]; then
+          exit 1;
```

Comment:
If files were changed, this currently fails without any indication as to *why* it failed:
![image](https://user-images.githubusercontent.com/8530778/185519717-00d01292-b432-4653-af91-08793e3310e8.png)
Would be a good idea to add a short error message here, similar to https://github.com/DisnakeDev/disnake/blob/c789800d32fdd227efb2755bc9ca9b973c2ab249/.github/workflows/changelog.yaml#L39

Reply:
https://github.com/DisnakeDev/disnake/pull/708/commits/67635b9f2565aeed32924ae68912aea33c67be54